app/(auth)/signin/page.tsx

```diff
@@ -57,32 +66,109 @@ export default function SignIn() {
   })
 
   return (
-    <>
-      <Heading>Welcome Back!</Heading>
-      <Heading>Sign In</Heading>
-      <Box as="form" onSubmit={signInHandler}>
-        <Flex flexDirection={'column'}>
-          <FormControl isInvalid={!!errors.email}>
-            <FormLabel>Email address</FormLabel>
-            <Input {...register('email')} />
-            {errors.email && (
-              <FormErrorMessage>{errors.email.message}</FormErrorMessage>
-            )}
-          </FormControl>
-          <FormControl isInvalid={!!errors.password}>
-            <FormLabel>Password</FormLabel>
-            <Input type="password" {...register('password')} />
-            {errors.password && (
-              <FormErrorMessage>{errors.password.message}</FormErrorMessage>
-            )}
-          </FormControl>
-          <PrimaryButton isLoading={isLoading} type={'submit'}>
-            Sign In
-          </PrimaryButton>
-          <Heading>{"Don't"} have an account yet?</Heading>
-          <Link href="/signup/email">Sign Up</Link>
-        </Flex>
-      </Box>
-    </>
+    <Box
```

Comment:
I personally think it's better to make a component for Fom because page.tsx's main job is routing. I usually don't put lots of jsx in page files for easier maintenance.

Reply:
True. I actually made three components and imported them into the page.tsx like this below. Let me know what you think. 


```typescript
<Card
          position="absolute"
          top={{ base: '0%', md: '50%' }}
          left="50%"
          transform={{
            base: 'translate(-50%, 0%)',
            md: 'translate(-50%, -50%)'
          }}
          bg="white"
          boxShadow={{ base: '0', md: '0 4px 16px rgba(0, 0, 0, 0.1)' }}
          maxW={{ base: '100%', md: '542px' }}
          py="40px"
          px={{ base: '16px', md: '80px' }}
        >
          <SignInHeader />
          <SignInForm />
          <SignInFooter />
 </Card>
```